python/text_replace.py

```diff
@@ -38,14 +40,15 @@
 
 SCRIPT_NAME    = "text_replace"
 SCRIPT_AUTHOR  = "xt <xt@bash.no>"
-SCRIPT_VERSION = "0.6"
+SCRIPT_VERSION = "0.7"
 SCRIPT_LICENSE = "GPL3"
 SCRIPT_DESC    = "Replaces text you write with replacement text"
 
 # script options
 settings = {
         'replacement_pairs': '(:=:),):=:(',   # pairs separated by , orig text and replacement separated by =
         'replacement_words': 'hhe=heh',       # words separated by , orig text and replacement separated by =
+        'replacement_prefixes': ':=/',       # strings separated by , orig prefix and replacement separated by =
```

Comment:
Shouldn't this value be empty by default, so that there are no changes for existing users?
Not sure if this default value is OK for most users.